walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -11,13 +11,79 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 package com.google.sps;
-
+ 
+import static com.google.common.collect.ImmutableList.toImmutableList;
+ 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Streams;
+import java.util.stream.Collectors;
+import java.util.HashMap;
 import java.util.Collection;
-
+import java.util.Collections;
+import java.util.ArrayList;
+ 
 public final class FindMeetingQuery {
+  private int nextAvailableStart = TimeRange.START_OF_DAY;
+
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<TimeRange> availableTimes = new ArrayList<TimeRange>();
```

Comment:
nit: defining it here doesn't seem necessary. Let's manually construct the (immutable) list when we return early.

Reply:
Done!